config/runtime.exs

```
@@ -40,7 +40,8 @@ if config_env() == :prod do
   # S3
   config :orbit, Orbit.S3,
     folders: [
-      glides_global: "s3://#{System.fetch_env!("GLIDES_BUCKET")}/global"
+      glides_global: "s3://#{System.fetch_env!("GLIDES_BUCKET")}/global",
+      rtr_public: System.get_env("S3_RTR_PUBLIC")
```

question: does this also need to be an S3 URI like `s3://...`?

---

suggestion: [fetch_env!/1](https://hexdocs.pm/elixir/1.18.3/System.html#fetch_env!/1) raises instead of returning nil, which we may want here (like the line above)

---

> question: does this also need to be an S3 URI like `s3://...`?

I reverse-engineered this from how Glides handles it, so the variable definition in the orbit [terraform_module](https://github.com/mbta/terraform_modules/blob/main/terraform-aws-mbta-orbit/main.tf#L44) formats the link in S3 URI format already

> suggestion: [fetch_env!/1](https://hexdocs.pm/elixir/1.18.3/System.html#fetch_env!/1) raises instead of returning nil, which we may want here (like the line above)

good suggestion, thanks! like the above, this is just me reverse-engineering from Glides (a little too exactly in this case)